package.json

```diff
@@ -5,7 +5,9 @@
     "author": "Deriv",
     "dependencies": {
         "@artsy/fresnel": "^3.2.1",
+        "@deriv/analytics": "^1.0.2",
```

Comment:
@markwylde-deriv Did we publish `@deriv/analytics` on `npm`? 🤔

Reply:
https://www.npmjs.com/package/@deriv/analytics

Reply:
So can we remove the local package in deriv-app repository and use the npm package? If we keep it then in less than a month these two packages will be inconsistent and go different path 😅

Reply:
Yeaah I was planning to do that as well, I've asked ako to create a card for that